corehq/apps/reports/models.py

```diff
@@ -760,6 +760,9 @@ def send(self):
             self.delete()
             return
 
+        if self.owner.is_deleted():
+            return
```

Comment:
I thought about that, but decided it would be nice if a user's saved reports were still available if they were restored.  Also, since this is the first time I've seen this error, this particular workflow must be pretty rare and deleting a few saved reports wouldn't have much of a performance impact.